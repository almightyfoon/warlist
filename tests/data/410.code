c2501b_-0E1l22223D3p3B2_2v2v2jfV2jfV___-0de722223I243qe41G1G3C3t3t2_2_3j