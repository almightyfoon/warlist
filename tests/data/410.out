https://conflictchamber.com/#c2501b_-0E1l2222m03p3B2_2v2v2jfV2jfV___-0de722223I24mee41G1G3C3t3t2_2_m2

Protectorate Army - 74 / 75 points
[Theme] Exemplar Interdiction

[Vindictus 1] Vice Scrutator Vindictus [+29]
 - Reckoner [16]
 - Reckoner [16]
High Exemplar Gravus [8]
Knights Exemplar Seneschal [0(5)]
Wrack [1]
Choir of Menoth (min) [4]
Exemplar Vengers (max) [20]
Exemplar Vengers (max) [20]
Knights Exemplar [9]
 - Knights Exemplar Officer [0(4)]
Knights Exemplar [9]
 - Knights Exemplar Officer [0(4)]

Protectorate Army - 75 / 75 points
[Theme] The Creator's Might

[Reznik 2] Servath Reznik, Wrath of Ages [+29]
 - Reckoner [16]
 - Reckoner [16]
 - Scourge of Heresy [16]
 - Templar [15]
 - Hierophant [0(3)]
Initiate Tristan Durant [0(4)]
 - Redeemer [11]
 - Redeemer [11]
The Covenant of Menoth [0(4)]
Vassal Mechanik [1]
Vassal Mechanik [1]
Choir of Menoth (min) [4]
Choir of Menoth (min) [4]
Visgoth Juviah Rhoven & Honor Guard [9]
